### PR TITLE
bug fix for raw protocol

### PR DESCRIPTION
**Pull request by shivam05011996**
Connected to #78.

### CODE REVIEW

**Comment by shivam05011996** [comment]
Is changing the version number required?

File: kwikapi/protocols.py [hunk 1]
```diff
@@ -35,6 +35,18 @@ def get_record_separator(self):
...
 
+    @staticmethod
+    def should_wrap():
+        '''
+        While returning the response the,
+        kwikapi will wrap the response as -
+        {success: value, result: value}
+
+        This is applicable to all the protocols,
+        except the raw protocol
```
Comment (prashanthellina) [inline kwikapi/protocols.py]: @shivam05011996 This line here about raw protocol is not necessary. Later if we add another proto that also doesn't need wrapping, this statement will no longer be true.

**Review by prashanthellina — APPROVED** [review]
@shivam05011996 approving this. just make the single changed requested. thanks a lot!

**Review by arrangeesh — APPROVED** [review]
looks good !

File: kwikapi/protocols.py (hunk 1 above)
Comment (shivam05011996) [reply kwikapi/protocols.py]: I have modified the docstring, such that, it will remain generalized for future protocol inclusions.

**Pull request merged**